src/server/socket.js

```diff
@@ -6,6 +6,9 @@ import { opponent as opponentOf } from '../common/util/game';
 
 import MultiplayerServerState from './multiplayer/MultiplayerServerState';
 
+const QUEUE_INTERVAL_MSECS = 500;
```

Comment:
This seems like a long wait for polling, no?

Reply:
I think it's fine, maybe we can change it later based on player feedback.